infrastructure/validators/operational-tasks/monitor-node.md

```diff
@@ -0,0 +1,371 @@
+```
+
+### Setup Gmail
+
+To use Gmail, you must generate an `app password` in your Gmail account to allow `AlertManager` to send you alert notification emails. Visit Google support for more details on setting up your [`app password`](https://support.google.com/accounts/answer/185833?hl=en){target=\_blank}. 
+
+Once you complete the setup, you should see something like the following:
+
+![grafana-am-1](/images/infrastructure/validator/operational-tasks/1-alert-manager.webp)
+
+Be sure to backup this password in a secure location.
+
+### Configure AlertManager
+
+There is a configuration file named `alertmanager.yml` inside the directory that you unzipped in the first step, but that isn't of use. Create a custom `alertmanager.yml` file under `/etc/alertmanager` with the following configuration:
+
+??? tip "Change `alertmanager` ownership"
+    Ensure to change the ownership of `/etc/alertmanager` to `prometheus` by executing the following command:
+    ```bash
+    sudo chown -R prometheus:prometheus /etc/alertmanager
+    ```
+
+```
+--8<-- 'code/infrastructure/validators/operational-tasks/monitor-node/alert-manager.yml'
+```
+
+With the preceding configuration, alerts will be sent using the email you set preceding. Remember to replace `YOUR_EMAIL` with the email where you want to receive alerts and paste the app password you saved earlier to the `YOUR_APP_PASSWORD`.
+
+Next, create another `systemd` configuration file named `alertmanager.service` by running the following command:
+``` bash
+sudo nano /etc/systemd/system/alertmanager.service 
+```
+Add the following code to the configuration file you just created:
+
+???tip  "SERVER_IP"
+    Change to your host IP address and make sure port 9093 is opened
+
+``` bash
+--8<-- 'code/infrastructure/validators/operational-tasks/monitor-node/systemd-alert-config.md'
+```
+
+To start the `Alertmanager`, run the following commands:
+
+```
+sudo systemctl daemon-reload && sudo systemctl enable alertmanager && sudo systemctl start alertmanager && sudo systemctl status alertmanager
+```
+
+```
+● alertmanager.service - AlertManager Server Service
+   Loaded: loaded (/etc/systemd/system/alertmanager.service; enabled; vendor preset: enabled)
+   Active: active (running) since Thu 2020-08-20 22:01:21 CEST; 3 days ago
+ Main PID: 20592 (alertmanager)
+    Tasks: 70 (limit: 9830)
+   CGroup: /system.slice/alertmanager.service
+```
+
+If you have configured the `AlertManager` properly, the `Active` field should display "active (running)."
+
+There is an `Alertmanager` plugin in Grafana that can help you monitor the alert information. To install the plugin, execute the followiing command:
+
+``` bash
+sudo grafana-cli plugins install camptocamp-prometheus-alertmanager-datasource
+```
+
+Once you successfully install the plugin, restart Grafana:
+
+``` bash
+sudo systemctl restart grafana-server
+```
+
+Now go to your Grafana dashboard `SERVER_IP:3000` and configure the `Alertmanager` datasource.
```

Comment:
@nhussein11 This whole section should be tested out and new screenshots. One of these screen caps is from 2018 so new ones are needed. Please follow this process to make sure the steps and commands are still accurate and take new screenshots that fit our guidelines (shows the browser, address bar, etc.). Thank you!